pkg/es/writer/esBulkHandler.go

```diff
@@ -405,6 +423,10 @@ func ProcessIndexRequest(rawJson []byte, tsNow uint64, indexNameIn string,
 		log.Errorf("ProcessIndexRequest: failed to add entry to in mem buffer, StreamId=%v, rawJson=%v, err=%v", streamid, rawJson, err)
 		return err
 	}
+	pleResponse := ple.GetResponse()
+	if pleResponse != "" {
+		return utils.TeeErrorf("ProcessIndexRequest: failed to add entry to in mem buffer, StreamId=%v, rawJson=%v, err=%v", streamid, rawJson, pleResponse)
```

Comment:
this will overwhelm the logs, we only want to return the error to the user that there was ingestion issue

Reply:
This function is now removed.